tests/server/server-executeBulkWrite-005.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+MongoDB\Driver\Server::executeBulkWrite() with write concern (replica set secondary, local DB)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS("REPLICASET"); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+$server = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY));
+
+/* The server ignores write concerns with w>2 for writes to the local database,
+ * so we won't test behavior for w=2 and w=majority. */
```

Comment:
@bjori: This is just something I stumbled upon while testing. I verified the behavior in the shell as well (3.0.x), so it's not a libmongoc thing. Not worth opening a server ticket for this IMO, as I imagine this is just the server ignoring an irrelevant option since `local` doesn't replicate.